### PR TITLE
WPC 679 remove tmp user

### PR DESCRIPTION
**Pull request by elias-summermatter**

### CODE REVIEW

**Comment by elias-summermatter** [comment]
@tomaszdurka Please review as asap, that i can merge it soon :-)  => I know the tests are not running but thats because the test not running on master too :-)

**Comment by elias-summermatter** [comment]
@michael-scheurer Can you have a look at my changes to your migration?

**Comment by elias-summermatter** [comment]
@tomaszdurka can you approve this pull request that i can merge it?

**Comment by elias-summermatter** [comment]
@tomaszdurka please can you chech?

File: packages/api-db-mongodb/src/migration.ts [hunk 1]
```diff
@@ -891,8 +895,19 @@ export const Migrations: Migration[] = [
     }
   },
   {
+    // Rename unused temp user collection
     version: 21,
-    async migrate(db, locale) {
+    async migrate(db) {
```
Comment (Itrulia) [inline packages/api-db-mongodb/src/migration.ts:900]: this should probably be v22 and not 21

**Review by Itrulia — CHANGES_REQUESTED** [review]

**Review by Itrulia — DISMISSED** [review]

File: packages/api-db-mongodb/src/migration.ts [hunk 2]
```diff
@@ -891,11 +895,22 @@ export const Migrations: Migration[] = [
...
     }
+  },
+  {
+    // Rename unused temp user collection
+   version: 22,
+   async migrate(db) {
+      const collections = await db.listCollections().toArray()
+      if (collections.includes('temp.users')) {
+        const tempUser = await db.collection('temp.users')
+        await tempUser.rename('temp.users.bak')
```
Comment (tomaszdurka) [inline packages/api-db-mongodb/src/migration.ts:911]: We don't need to migrate temp users to users at all?

**Review by tomaszdurka — APPROVED** [review]

**Review by Itrulia — APPROVED** [review]

**Review by Itrulia — APPROVED** [review]

**Review by tomaszdurka — APPROVED** [review]

**Pull request merged**